src/network/network-er-x-sfp.md

```diff
@@ -0,0 +1,332 @@
+set protocols babeld filter eth1 if eth1
+set protocols babeld filter eth1 type redistribute
+
+set protocols babeld filter eth2 action allow
+set protocols babeld filter eth2 if eth2
+set protocols babeld filter eth2 type redistribute
+
+set protocols babeld filter eth3 action allow
+set protocols babeld filter eth3 if eth3
+set protocols babeld filter eth3 type redistribute
+
+
+set protocols babeld filter eth4 action allow
+set protocols babeld filter eth4 if eth4
+set protocols babeld filter eth4 type redistribute
+
+set protocols babeld filter l2tpeth0 action allow
+set protocols babeld filter l2tpeth0 if l2tpeth0
+set protocols babeld filter l2tpeth0 type redistribute
+```
+
+## Gateway Loop issues
+
+The paradox of the gateway and routing babeld:
+
+- DHCP provides the default route to exit to the Internet
+- Tunnel is established by using the default route
+- Babeld receives the IP address from the neighbour on the other side of the tunnel
+- The default route gets replaced with the exit node
+- All Internet traffic is rerouted through the tunnel
+- Since the tunnel uses Internet traffic it is also rerouted
+- Tunnel collapses, since it is now feeding into itself
+
+
+### Workaround 1 - Static IP/Route
+
+- Define static IP on `ETH0` (WAN facing port)
+- Remove default gateway
+- Create route out to the gateway for only the exit node
+
+```
+delete interfaces ethernet eth0 address dhcp
+set interfaces ethernet eth1 address 192.168.2.254/24    
+
+delete protocols static route 0.0.0.0/0
+set protocols static route 199.195.250.209/32 next-hop 192.168.2.1
+```
+
+### Workaround 2 - Route Tables
+
+Route tables allow the use of the default route (0.0.0.0) on the node to still point to the local Internet connection. Babeld is instructed to place all the installed routes into a new routing table. A rule is installed to use this routing table for all packets coming in from interfaces we use babeld on. Then finally routes for the local node are added manually since babeld will not install them since they are local. Rules and routes must be installed in both the IPv4 and IPv6 stacks.
+
+```
+set protocols babeld export-table 10
+```
+
+Append the following lines to `/etc/rc.local` for each interface that babeld will route though. `-6` indicates it is an IPv6 address.
+
+```
+ip [-6] rule add iif <INT> table 10
+```
+
+Append the following lines for each IP address range your node is responsible for. `-6` indicates it is an IPv6 address.
+```
+ip [-6] route add <IPv4>/<CDIR> dev <INT> table 10
+```
+
+#### Example
+```
+ip rule add iif eth4 table 10
+ip -6 rule add iif eth4 table 10
+ip rule add iif l2tpeth63 table 10
+ip -6 rule add iif l2tpeth63 table 10
+
+ip route add 100.64.20.0/24 dev eth4 table 10
+ip -6 route add fd54:4f00::/24 dev eth4 table 10
+ip route 100.127.2.252/30 dev l2tpeth63 table 10
+ip -6 route add fd74:6f6d:7368:7f02::fc/126 dev l2tpeth0 table 10
+```
+
+## OpenVPN - Management Tunnel
```

Comment:
```suggestion
## OpenVPN - Management Tunnel

Management tunnel to allow an alternative means of accessing the router out side of the mesh network, in case the mesh fails.
```